core/lib/types/src/server_notification.rs

```diff
@@ -0,0 +1,17 @@
+use zksync_basic_types::L1BlockNumber;
+
+use crate::H256;
+
+#[derive(Debug, PartialEq)]
+pub enum GatewayMigrationState {
```

Comment:
is there any difference logic-wise between "NotStarted" and "Finalized"? Maybe it makes to change it to `InProgress` and `NotInProgress`

Reply:
fixed